Week3/script.js

```diff
@@ -146,13 +145,13 @@ console.log(vehicles[2]);
 function newVehicle(color,code,age) {
   let type,condition = "";
   if (code === 1) {
-    type = " motorbike";
+    type = vehicles[0];
   } if (code === 2) {
-    type = " caravan";
+    type = vehicles[1];
   }  if (code === 3) {
-    type = " bike";
+    type = vehicles[2];
   } if (code === 4) {
-    type = " car";
+    type = vehicles[3];
```

Comment:
Ah, do you see a `pattern` here? Now you changed the code to this:

```
  if (code === 1) {
    type = vehicles[0];
  }
  if (code === 2) {
    type = vehicles[1];
}
..
```

The `index` of the array that you use, is always `1` value lower than the value checked in your `if-statement`. Do you think you can change the code so that you can use the value in `code` and use it as an `index` for the list?